src/participant/sampleDocuments.ts

```diff
@@ -0,0 +1,31 @@
+import type { Document } from 'bson';
+import { toJSString } from 'mongodb-query-parser';
+
+const MAX_ARRAY_LENGTH_OF_SAMPLE_DOCUMENT_VALUE = 3;
+
+const MAX_STRING_LENGTH_OF_SAMPLE_DOCUMENT_VALUE = 20;
+
+export function formatSampleDocuments(obj: Document) {
+  return toJSString(format(obj));
+}
+
+function format(obj: Document) {
```

Comment:
Not sure what the coding style of the codebase is, but should we add some type declarations for the arguments and function returns?

Reply:
It should have a return type, but sometimes we forget :) I will add `@typescript-eslint/explicit-function-return-type` to catch those cases.